source/_includes/installation/operating_system.md

```diff
@@ -183,6 +185,8 @@ Use this method only if Method 1 does not work for you.
 
     {% if variant.key == "odroid-n2" %}
     [Guide: Flashing Odroid-N2 using OTG-USB](/hassio/flashing_n2_otg/)
+    {% elsif variant.key == "odroid-m1s" %}
+    [Guide: Flashing Odroid-M1S using OTG-USB](/hassio/flashing_m1s_otg/)
```

Comment:
Guide: Boot Odroid-M1S into UMS

Reply:
Could you explain why you might prefer that title? I ask because from my point of view the procedure is the same as for the N2+. So from this perspective it is just consistent to also have the same title.
Or the other way: wouldn't your title proposition also apply to the N2+? If yes: why not keeping it the same for the sake of consistency?

Reply:
I suppose that doesnt apply if your not referring back to these steps after.. Keep it the same as N2 if you prefer

Reply:
> I suppose that doesnt apply if your not referring back to these steps after.. Keep it the same as N2 if you prefer

What do you mean by that? Do you agree to @darkxst to change the guides title? Or do you think that keeping the name identical to the N2 stuff is okay because the following steps / the content is basically the same?

Reply:
sorry that was my comment also, but wrong account! Keep it the same as N2 if you prefer.

Reply:
Alright, than I would leave it as it currently is. Maybe it is a matter of personal opinion if one prefers cross references over duplicated content or vice versa. If you are fie with the current state I mark this conversation as resolved.

Reply:
Ok, LGTM.

You could probably add the board to this list also:
https://github.com/home-assistant/home-assistant.io/blob/33771b9780fac2b4db8ba6f3edb96a3217db36ce/source/_includes/installation/operating_system.md#suggested-hardware

Reply:
> You could probably add the board to this list also: https://github.com/home-assistant/home-assistant.io/blob/current/source/_includes/installation/operating_system.md#suggested-hardware

Well, yes and no. I was aware of that page and thought about that, too. But the section is written from the perspective of the HA administration with the wording "we recommend". This is not the position I am in. Additionally a device support will not automatically mean a recommendation (like the C2, that is supported but not explicitly recommended).
So I think to list the device under "recommendations" it will require either more user experiences or objective benchmarks as a HA device to officially recommend it.
Apart from that I would have to create affiliate links, that I just can't. AND: ameridroid seems not to have the M1S in their catalog at all (at the time of writing).

To sum it up:
- mention in the download section: yes (done)
- mention in recommendations: not yet possible
- add to list of affiliate links: not possible (for me) due to not available in online shop

What do you think?

Reply:
Yeh they do have it:
https://ameridroid.com/products/odroid-m1s

> Additionally a device support will not automatically mean a recommendation (like the C2, that is supported but not explicitly recommended).

Right if it was a completely new board/soc could be different, but since its only slightly modified from M1, support should be as good as that from day one. C2 is not a good comparison, it is obsolete and discontinued! Anyway see what @sairon thinks for this.

Reply:
> Yeh they do have it: https://ameridroid.com/products/odroid-m1s
> 
Oh, my bad! I went through the SBC sub menu of ameridroid and did not find it. Maybe I should have tried the text search...

Reply:
Done